tests/src/roomHandlers/exampleRoom1Handler.test.ts

```diff
@@ -0,0 +1,32 @@
+import { expect } from 'chai';
+import { testSingletonModule } from '../../testHelpers/semanticMocha';
+import { ExampleRoom1Handler } from '../../../src/roomHandlers/exampleRoom1Handler';
+import { CommandResult, RoomState } from '../../../src/utils/types';
+
+testSingletonModule(
+  'roomHandlers/ExampleRoom1Handler',
+  ({ testIntegration }) => {
+    testIntegration('run', ({ testScenario }) => {
+      testScenario('with the "leave" command')
```

Comment:
Should there be a test case for a "non leave" command?

Reply:
I think we should discuss our possibilities for testing this

- Should we add test coverage for the DEFAULT_COMMAND, and an invalid command to all room handlers?
- Should we make these tests unit tests? (I would like to talk about the specifics of this point)
- Should the state transition test helper I'm making prove pre-built scenarios to handle this?